test/document_stores/test_elasticsearch.py

```diff
@@ -31,9 +31,11 @@ def ds(self):
             host=os.environ.get("ELASTICSEARCH_HOST", "localhost"),
             create_index=True,
         )
+        ds.delete_documents()
+        ds.delete_labels()
         yield ds
-        ds.delete_index(self.index_name)
-        ds.delete_index(labels_index_name)
+        ds.delete_documents()
+        ds.delete_labels()
```

Comment:
This change is to let developers use local ES installations that might have some data already